### DISCUSSION TITLE
How to learn writing own custom patches?

### DISCUSSION THREAD

**Discussion by yc-codes** [ReVanced/revanced-discussions#629] [Question]
Hello there,

Thanks for your work guys, really appreciate it,

A simple question

How to learn what you guys are writing like,
I could understand the code in kotlin but how to find the codes that you override

please write some docs about it, or can share?

Thanks 🙏

**Comment by GeekCornerGH** [comment]
That's called reverse engineering, that's it

**Reply by oSumAtrIX** [reply]
Also looking into existing patches is a good source.

**Comment by ghost** [comment]
If you're enough delusional to learn bytecode, you'll be able to create your own patches (because them must be created before written and pushed on this repo).
